doc/source/development/contributing_environment.rst

```diff
@@ -249,6 +249,11 @@ To compile pandas with setuptools, run::
 
    python setup.py develop
 
+.. note::
+   If pandas is already installed, you have to uninstall it first::
+
+        pip uninstall pandas
```

Comment:
Rather than raw pip, should you use `python -m pip uninstall pandas`, which seems to be more consistent with the rest of the page?

Reply:
done